service-base/src/main/java/org/eclipse/hono/service/management/tenant/Tenant.java

```diff
@@ -138,4 +139,19 @@ public Tenant putExtension(final String key, final Object value) {
         this.extensions.put(key, value);
         return this;
     }
+
+    /**
+     * Adds the specified adapter to the list of adapters for this Tenant.
+     * 
+     * @param adapter The adapter to add for the tenant.
+     * @return        This instance, to allow chained invocations.
+     */
+    public Tenant addAdapter(final Adapter adapter) {
```

Comment:
actually, this method does not add an adapter but an adapter *configuration*. As such, I think the method should be called accordingly, e.g. *addAdapterConfig*